Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [JASP/jaspDesktop](https://hosted.weblate.org/projects/jasp/jaspdesktop/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/jasp/jaspdesktop/horizontal-auto.svg)

ertugruluyar-github commented:
I think Turkish language support is enough for the desktop app, I was added some missing things and checked.

shun2wang commented:
> I think Turkish language support is enough for the desktop app, I was added some missing things and checked.

Thanks, added in https://github.com/jasp-stats/jasp-desktop/pull/5991

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (1205 of 1205 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/pt/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (1205 of 1205 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/pt/
